### DISCUSSION TITLE
Which one should be used? readBody or decodeBody

### DISCUSSION THREAD

**Discussion by borsemayur2** [alephjs/aleph.js#163] [Q&A]
`types.ts` file in Github repo mentions `readBody` while deno module and documentation mentions `decodeBody`. 

decodeBody is working while `readBody` gives error `request.readBody is not a function`.

Versions:
Deno: 1.7.5
Aleph.js: v0.3.0-alpha.8

**Comment by shadowtime2000** [comment]
I am fairly certain it is `decodeBody`. I wasn't able to pay much attention to the discussion to add that feature, so maybe @shinspiegel can tell us what happened.

**Reply by ije** [reply]
sorry I changed it to `readBody`, the docs is still in 0.2.28

**Reply by ije** [reply]
and I am launching the beta version, will update the website using 0.3
